unitTests/test_enforce.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include "gtestwrapper.h"
 
+#include <stdexcept>
```

Comment:
Why on earth did this start breaking?! This file was touched last on March 11th. Computers are sometimes weird.

Reply:
I see two possibilities:
- We changed some of our public headers and removed such inclusion from some places.
- The internal STL files in CentOS changed for some reason.

Anyways, it is good to have the GitLab CI jobs to detect such issues :grinning: